fail2ban/tests/sockettestcase.py

```diff
@@ -83,3 +90,17 @@ def testSocketForce(self):
 		self.server.stop()
 		serverThread.join(1)
 		self.assertFalse(os.path.exists(self.sock_name))
+
+
+class ClientMisc(unittest.TestCase):
+
+	def testPrintFormattedAndWiki(self):
+		# redirect stdout to devnull
+		saved_stdout = sys.stdout
+		sys.stdout = open(os.devnull, 'w')
+		try:
+			protocol.printFormatted()
+			protocol.printWiki()
+		finally:
+			# restore stdout
+			sys.stdout = saved_stdout
```

Comment:
FWIW, someone should adopt smth like our lightweight but thorough mocking for testing stdout/stderr in datalad:  https://github.com/datalad/datalad/blob/master/datalad/utils.py#L213  (and probably swallow_logs for a pair). Becomes as neat as

```python
    with swallow_outputs() as cmo:
        with swallow_logs(new_level=logging.INFO) as cml:
            ret = runner.run(cmd, log_stderr=False)
            eq_(cmo.err.rstrip(), "stderr-Message should not be logged")
            eq_(cml.out, "")
```
don't remember if they are fully 2.6 compatible